Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -283,6 +262,48 @@ def validate_data_and_labels(data, exist_labels_counter, labels_mapping, missing
         return_error('\n'.join(err))
 
 
+def return_file_result_with_predictions_on_test_set(data, original_text_fields, test_index, text_field, y_test, y_pred_dict):
+    if original_text_fields is None or original_text_fields.strip() == '':
+        original_text_fields = [text_field]
+    else:
+        original_text_fields = re.split(r'[|,]', original_text_fields)
+    predictions_data = {}
```

Comment:
add the tokenized text field in any case, even if the origianl text fields specified